webapplication/requirements.txt

```diff
@@ -8,6 +8,7 @@ django-admin-rangefilter==0.6.3
 drf-yasg==1.20.0
 uWSGI==2.0.18
 coverage==6.3.2
+Pillow==9.5.0
```

Comment:
Why is this needed?

Reply:
ERRORS:
aoi.Component.description_picture: (fields.E210) Cannot use ImageField because Pillow is not installed.
        HINT: Get Pillow at https://pypi.org/project/Pillow/ or run command "python -m pip install Pillow".
 *  Terminal will be reused by tasks, press any key to close it.